### ISSUE TITLE
PopupMenuItem is not recognized as a button

### ISSUE THREAD

**Issue by mehmetf** [flutter/flutter#45044]
Internal: b/144454022

TalkBack on Android is not recognizing a `PopupMenuItem` like
```dart
PopupMenuItem<WhyFarther>(
            value: WhyFarther.harder,
            child: Text('Working a lot harder'),
)
```
to be a `Button`.

Other buttons in the toolbar are read as `<text>, Button` with perhaps the state "disabled" appended.

A simple workaround is to wrap the child with Semantics like:
```dart
PopupMenuItem<WhyFarther>(
            value: WhyFarther.harder,
            child: Semantics(
                 child: Text('Working a lot harder'),
                 button: true,
            ),
)
```

**Comment by chunhtai** [comment]
Why does the PopupMenuItem need to be consider as button? We don't use the term `button` to refer `PopupMenuItem` in our api documentation. It should be treated as regular widget that has tap gesture recognizer. The developers can customize the semantics flag based on their use case, but i don't think we should provide an opinion on that.

**Comment by mehmetf** [comment]
It could be because that's what Android does in native apps. You can ask for clarification in the linked buganizer issue.

**Comment by chunhtai** [comment]
There are two issues with current semantics of `PopupMenuItem`

1. The disabled `PopupMenuItem` does not pronounce disabled in semantics talk back
2. The `PopupMenuItem` is not treated as button in Semantics talk back

I think 1 is definitely something we should fix. For 2, the customer think PopupMenuItem is usually used in toolbar and should be treated as button. I think that makes sense, I will go ahead and implement the fix.

**Comment by chunhtai** [comment]
post commit test failure
